core/controllers/acl_decorators.py

```diff
@@ -487,6 +487,44 @@ def test_can_manage_account(self, **kwargs):
     return test_can_manage_account
 
 
+def can_manage_own_profile(handler):
```

Comment:
Are you planning to hook this up in subsequent PRs?

Reply:
I will use this decorator in subsequent PRs to implement controllers for profile and there itself I will also see if additional tests are needed here.
Is this what you wanted to know? Didn't get the question properly.

Reply:
Yes, I just wanted to be clear that I understood the plan.